gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFSInputStream.java

```diff
@@ -182,16 +194,24 @@ public synchronized int read(byte[] buf, int offset, int length) throws IOExcept
   @Override
   public synchronized int read(long position, byte[] buf, int offset, int length)
       throws IOException {
-    Stopwatch stopwatch = Stopwatch.createStarted();
-
-    int result = super.read(position, buf, offset, length);
-    readAPITrace(POSITIONAL_READ_METHOD, stopwatch, position, offset, length, result, Level.FINE);
-    if (result > 0) {
-      // -1 means we actually read 0 bytes, but requested at least one byte.
-      statistics.incrementBytesRead(result);
-      totalBytesRead += result;
-    }
-    return result;
+    return GhfsStorageStatistics.trackDuration(
+        storageStatistics,
+        GhfsStatistic.STREAM_READ_OPERATIONS,
```

Comment:
Keeping parity with what is there in Master. Additional metrics can be added later if required.

Reply:
Thinking more about this, I think the current logic will double count since super.read() will result in a call back to read(buffer, offset, length). Removed the tracking for positional read.